custom-namespaces.yaml

```
@@ -585,6 +585,10 @@ firefox_accounts:
       type: table_view
       tables:
         - table: moz-fx-data-shared-prod.accounts_frontend_derived.login_engagement_funnel_v1
+    recovery_phone_segments:
+      type: table_view
+      tables:
+        - table: moz-fx-data-shared-prod.accounts_backend_derived.monitoring_db_recovery_phones_counts_v1
```

These auto-generated views don't get any measures by default, so you might want to define measures here (like was done for `monitoring_db_counts`):
```suggestion
        - table: moz-fx-data-shared-prod.accounts_backend_derived.monitoring_db_recovery_phones_counts_v1
      measures:
        total_rows_sum:
          type: sum
          sql: ${total_rows}
```

---

added

---

thank you